ext/xvalidator/validator.go

```diff
@@ -133,6 +133,14 @@ func New(prefix string) (*validator.Validate, ut.Translator) {
 		return t
 	})
 
+	validate.RegisterValidation("bigint", IsBigInt)
+	validate.RegisterTranslation("bigint", trans, func(ut ut.Translator) error {
+		return ut.Add("bigint", "{0} must be a valid credit", false)
```

Comment:
There is not about credit here.
```suggestion
		return ut.Add("bigint", "{0} must be a valid big int", false)
```